src/gsy_e/models/area/__init__.py

```diff
@@ -373,6 +373,8 @@ def cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fal
         else:
             changed_balancing_market = None
 
+        self._markets.update_area_market_id_lists()
```

Comment:
I have the feeling that we may end up out of sync between the markets ids lists and the actual markets in this _markets handler
I think a better solution would be to call this `update_area_market_id_lists` at the end of each `create_new_*_market` method call, this will also allow us to make `update_area_market_id_lists` private

Reply:
Good idea! I was also a bit insecure here. Your solution seems more secure.

Reply:
Actually, now I remember, why I decided to to it like this: If we call it multiple times per market_slot, it will add an unnecessary load overhead. So, I would vote for keeping it like this.

Reply:
How often do we access these market ids lists? we can create properties for them which will ensure that they're always up to date (and no computation will be done unless we do access the property)

Reply:
They are access on every market event and area event (so on every tick, market_cycle, offer, bid, accept_offer, etc) So a LOT of times.

Reply:
We may still cache them, however let's leave it for now